.github/workflows/pre-merge.yml

```diff
@@ -1,14 +1,16 @@
-name: Build and Test
+name: Build and Test Unstable
 on:
   pull_request:
     branches:
-      - master
-      - release/*
+      - unstable
+  schedule:
```

Comment:
Maybe a separate workflow that is scheduled - specifically for unstable is better?